x/pocketcore/types/service.go

```diff
@@ -240,7 +243,8 @@ func (rr RelayResponse) Hash() []byte {
 		Proof:     rr.Proof.HashString(),
 	})
 	if err != nil {
-		panic(fmt.Sprintf("an error occured hashing the relay response:\n%v", err))
+		fmt.Println(fmt.Errorf("an error occured hashing the relay response:\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured hashing the relay response:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)